### PR TITLE
Editing - Deploying ECK on OpenShift

### PR DESCRIPTION
**Pull request by alaudazzi**
Adjusted para setting and added minor edits.

### CODE REVIEW

File: docs/openshift.asciidoc [hunk 1]
```diff
@@ -5,16 +5,18 @@ This section explains how to run ECK on OpenShift.
...
+Before deploying an Elasticsearch cluster with ECK, make sure you correctly applied the `vm.max_map_count` setting on all the nodes of your cluster. Pods created by ECK are likely to run with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Secure Context Constraints] (SCC), they run with a limited set of privileges and cannot change this setting on the nodes that host them. For more details, see the https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[Virtual memory].
```
Comment (pebrc) [inline docs/openshift.asciidoc]: Should the link at the end somehow make clear that it is not part of the ECK docs but Elasticsearch documenation?

Comment (alaudazzi) [reply docs/openshift.asciidoc]: Would that be OK?

> For more details, see the Elasticsearch documentation on https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[Virtual memory].

Comment (pebrc) [reply docs/openshift.asciidoc]: I like it 👍

Comment (anyasabo) [inline docs/openshift.asciidoc]:
```suggestion
Before deploying an Elasticsearch cluster with ECK, make sure you correctly applied the `vm.max_map_count` setting on all the nodes of your cluster. Pods created by ECK are likely to run with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Secure Context Constraints] (SCC): they run with a limited set of privileges and cannot change this setting on the nodes that host them. For more details, see the https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[Virtual memory].
```

nit: punctuation change

**Pull request merged**